cloud/spawn.go

```diff
@@ -138,6 +141,17 @@ func CreateSpawnHost(ctx context.Context, so SpawnOptions, settings *evergreen.S
 			return nil, errors.Errorf("cannot use volume in zone '%s' with host in region '%s'", volume.AvailabilityZone, so.Region)
 		}
 	}
+	if so.UseProjectSetupScript {
+		so.ProvisionOptions.SetupScript, err = model.GetSetupScriptForTask(ctx, so.ProvisionOptions.TaskId)
```

Comment:
Does this mean that the project-level script runs instead of the distro setup script? Will that cause problems?

Reply:
No, the setup script in provision options is just for the user-defined setup script. The distro setup script isn't stored here.

Reply:
So before we used SetupScript to hold the user-defined setup script only, now we might store the project-level script there (which is fine bc they're mutually exclusive).